src/Modal.js

```diff
@@ -440,12 +456,21 @@ class Modal extends React.Component {
     }
   }
 
+  handleDocumentKeyDown = (e) => {
+    if (this.props.keyboard && e.key === 'Escape' && this.isTopModal()) {
+      if (this.props.onEscapeKeyDown) {
+        this.props.onEscapeKeyDown(e);
+      }
+
+      this.props.onHide();
+    }
+  }
+
   handleDocumentKeyUp = (e) => {
-    if (this.props.keyboard && e.keyCode === 27 && this.isTopModal()) {
+    if (this.props.keyboard && e.key === 'Escape' && this.isTopModal()) {
```

Comment:
This is a small question I could probably google and get the answer but maybe someone already knows. Will this key always be `'escape'` even on a Chinese keyboard for example? Just concerned about localization when hard coding english words in the codebase.

Reply:
Yup, see https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values

(Note `Escape`, never `escape` 😛)